docs/demo.rst

```diff
@@ -59,8 +59,8 @@ Now we can download some example data, which we'll use for our demo:
 
 .. code-block:: bash
 
-    (ga4gh-env) $ wget http://www.well.ox.ac.uk/~jk/ga4gh-example-data-v2.1.tar
-    (ga4gh-env) $ tar -xvf ga4gh-example-data-v2.1.tar
+    (ga4gh-env) $ wget http://www.well.ox.ac.uk/~jk/ga4gh-example-data-v3.0.tar
```

Comment:
Excellent idea @dcolligan. It turns out we can add some rst substitutions easily enough using the ``rst_prolog`` in conf.py. These work perfectly well in the main document. However, they don't work in code blocks, which is the only place we actually want them. We can work around this by using parsed-literal blocks:

http://stackoverflow.com/questions/27392934/sphinx-variable-substitution-in-code-blocks

However, these look totally different to the rest of the code-blocks and the result is very jarring visually. It would be great to solve this problem properly at some point, but I think we'll have to leave it as it is for now.